requirements-test.txt

```diff
@@ -4,5 +4,5 @@ isort
 flake8
 # we need this otherwise
 # the travis build may break
-cryptography==2.1.2
-pyopenssl>=17.1.0,<17.4.0
+cryptography>=2.1.4
+pyopenssl>=17.5.0
```

Comment:
add `,<17.6.0`which should protect us against new backward incompatible changes released in the future